node/node_test.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/rollkit/rollkit/types"
 )
 
-var MockServerAddr = ":7980"
+var MockServerAddr = "http://localhost:7980"
```

Comment:
this same value is reference directly in tests. we should just reuse it. If we need to extract it to avoid dependency loops we can put it in test/mocks

Reply:
Moved consts to `test/mocks`.

Reply:
We need these consts per pkg, so moved them out of `test/mocks` back into each pkg.

Also `test/mocks` contains autogenerated mocks was probably not a good place anyway.